web/src/engine/FetchProviderNodeWebkit.ts

```diff
@@ -158,7 +158,7 @@ export async function FetchCSS<T extends HTMLElement>(request: FetchRequest, que
     return [...dom.querySelectorAll(query)] as T[];
 }
 
-export async function FetchGraphQL<TResult>(request: FetchRequest, operationName: string, query: string, variables: string): Promise<TResult> {
+export async function FetchGraphQL<TResult>(request: FetchRequest, operationName: string, query: string, variables: unknown): Promise<TResult> {
```

Comment:
Using `unknown` is quite an anti-pattern for using type definitions.
Consider using an appropriate type or add JSDoc comment for the method parameter.